packages/contracts/addresses-testnet.txt

```diff
@@ -0,0 +1,17 @@
+0xa29cD06D675179c9886418E2D8506AF5BA9cdBFB
+0x29b8B2D4b19c80C745D07fECA5782e7f91e570E8
+0x0DC00c3d8aE40A91C4F19f9B050a5b66503cfaaA
+0xB529Ba3866f31d30922b3149ebdb7F4df6De0A43
+0x8ea64941C9d1ccf3689f360BeE434610f5eE6C9E
+0xdB2F7bC032D1d35E6C67e9077D1F39A4999774E1
+0x1c3A9ff56c57e06b8eC95f9B4a79E5fE8E5E0B37
+0x72C4869aad514D30f54f79fBD2Cf7E16e34b0956
+0xe401eA8E74a58C3Bf177e2E31D11DFE6dEb452e3
+0xb683f8bCcF83B96da3e125A302d1DdB1cb4103D2
+0x99CD3D7c1492010e4E344FE34f7ae611Ff5d9Fd9
+0x886f3BFc4C1d3CfCF4E0AE3EC91b0E238359b2d8
+0xb9851Eb6062950B6c2A6C344729e0aB1f93AFCef
+0x8Ec6fB913c864404a4FA07fF6c348F79841a99d8
+0x0848F3700754cb12D5620556618Dab59F5542E64
+0xeA1BA2C23EC79F4C8882947c0Aa6eA5992059061
+0x29764feFbac6011d485904511b7e5282208FAECB
```

Comment:
change to pre address for `bittesttoken`: 0x5a94dc6cc85fda49d8e9a8b85dde8629025c42be